ServerCore/Pages/Events/Map.cshtml.cs

```diff
@@ -37,7 +37,7 @@ public async Task<IActionResult> OnGetAsync(int? refresh)
 
             if (EventRole == EventRole.admin)
             {
-                puzzles = await _context.Puzzles.Where(p => p.Event == Event)
+                puzzles = await _context.Puzzles.Where(p => p.Event == Event && p.IsPuzzle)
```

Comment:
I found this useful during Hunt when I needed to see how timed unlocks were running. Could they just sort to the end?

Reply:
Not a bad idea, will fix